From 16d1576ab7166ee392f1b9d1843d2b674b38fc51 Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@rr-research.no>
Date: Sat, 17 Mar 2018 15:54:33 +0100
Subject: [PATCH] Fix QtTesting include dirs

---
 Libs/MRML/Widgets/CMakeLists.txt | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/Libs/MRML/Widgets/CMakeLists.txt b/Libs/MRML/Widgets/CMakeLists.txt
index c1e56e7b1..ae5e19f8b 100644
--- a/Libs/MRML/Widgets/CMakeLists.txt
+++ b/Libs/MRML/Widgets/CMakeLists.txt
@@ -32,6 +32,9 @@ set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${MRMLWidgets_SOURCE_DIR}/CMake")
 #
 set(CTK_LIBRARIES CTKVisualizationVTKWidgets)
 if(Slicer_USE_PYTHONQT)
+
+  find_package(QtTesting REQUIRED)
+
   list(APPEND CTK_LIBRARIES
     CTKScriptingPythonWidgets
     )
@@ -444,6 +447,7 @@ set(include_dirs
   ${MRMLCore_INCLUDE_DIRS}
   ${MRMLLogic_INCLUDE_DIRS}
   ${MRMLDisplayableManager_INCLUDE_DIRS}
+  ${QtTesting_INCLUDE_DIRS}
   )
 include_directories(${include_dirs})
 
-- 
2.16.1

