From cbf7616667602fb22638645c8cf77057678588c9 Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@rr-research.no>
Date: Wed, 14 Mar 2018 11:50:20 +0100
Subject: [PATCH] Fix C++ standard to C++11

---
 CMakeLists.txt | 13 +------------
 1 file changed, 1 insertion(+), 12 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index b40f608e6..e06d98e72 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -3,18 +3,7 @@ cmake_minimum_required(VERSION 3.5)
 #-----------------------------------------------------------------------------
 # Enable C++11
 #-----------------------------------------------------------------------------
-set(_msg "Setting C++ standard")
-message(STATUS "${_msg}")
-if("${Slicer_VTK_VERSION_MAJOR}" MATCHES "^(8|9)$" OR DEFINED Qt5_DIR)
-  set(CMAKE_CXX_STANDARD 11)
-  # Since SimpleITK requires CMP0067 to properly support C++11, requires
-  # the corresponding CMake version.
-  if(CMAKE_VERSION VERSION_LESS "3.8.2")
-    message(FATAL_ERROR "CMake >= 3.8.2 is required to enable C++11 support")
-  endif()
-else()
-  set(CMAKE_CXX_STANDARD 98)
-endif()
+set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
 message(STATUS "${_msg} - C++${CMAKE_CXX_STANDARD}")
-- 
2.16.1

